### PR TITLE
Added missing validation translations

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is a bugfix.
## Changelog

```markdown
### Added
- Added missing validation translations
- Added missing translation to blocks
```

## Subject

The translation was missing.

### CODE REVIEW

File: src/Resources/translations/validators.de.xliff [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file source-language="en" target-language="de" datatype="plaintext" original="validators.en.xliff">
+        <body>
+            <trans-unit id="sonata.block.menu.not_existing">
+                <source>sonata.block.menu.not_existing</source>
+                <target>Menu %name% ist nicht vorhanden.</target>
```
Comment (greg0ire) [inline src/Resources/translations/validators.de.xliff]: Maybe quote `%name%`?

**Comment by greg0ire** [comment]
https://github.com/core23/SonataBlockBundle/pull/1

**Comment by core23** [comment]
Merci!

**Comment by greg0ire** [comment]
You can squash all 3 commits

File: src/Block/Service/MenuBlockService.php [hunk 2]
```diff
@@ -172,16 +174,43 @@ protected function getFormSettingsKeys()
         $choiceOptions['choices'] = array_flip($this->menuRegistry->getAliasNames());
 
         return [
-            ['title', 'text', ['required' => false]],
-            ['cache_policy', 'choice', ['choices' => ['public', 'private']]],
+            ['title', 'text', [
+                'required' => false,
+                'label' => 'form.label_title',
+            ]],
+            ['cache_policy', 'choice', [
```
Comment (core23) [inline src/Block/Service/MenuBlockService.php]: We need to change this afterwards to class names

**Comment by core23** [comment]
@greg0ire Can you add the other translations too?

**Comment by greg0ire** [comment]
Aw shit I knew I forgot something

File: src/Resources/translations/validators.de.xliff [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file source-language="en" target-language="de" datatype="plaintext" original="validators.en.xliff">
+        <body>
+            <trans-unit id="sonata.block.menu.not_existing">
+                <source>sonata.block.menu.not_existing</source>
+                <target>Menü %name% ist nicht vorhanden.</target>
```
Comment (greg0ire) [inline src/Resources/translations/validators.de.xliff]: Please quote parameters

**Comment by greg0ire** [comment]
That's what I call using git in a truly decentralized way 😎

**Comment by greg0ire** [comment]
Please squash

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**